reports/gas-checker.txt

```diff
@@ -8,28 +8,28 @@
 ║                                │       │               │     (delta for │        (compared to ║
 ║                                │       │               │    one UserOp) │     account.exec()) ║
 ╟────────────────────────────────┼───────┼───────────────┼────────────────┼─────────────────────╢
-║ simple                         │     1 │         77911 │                │                     ║
+║ simple                         │     1 │         78098 │                │                     ║
 ╟────────────────────────────────┼───────┼───────────────┼────────────────┼─────────────────────╢
-║ simple - diff from previous    │     2 │               │          43377 │               12332 ║
+║ simple - diff from previous    │     2 │               │          43600 │               12555 ║
 ╟────────────────────────────────┼───────┼───────────────┼────────────────┼─────────────────────╢
-║ simple                         │    10 │        468533 │                │                     ║
+║ simple                         │    10 │        470511 │                │                     ║
 ╟────────────────────────────────┼───────┼───────────────┼────────────────┼─────────────────────╢
-║ simple - diff from previous    │    11 │               │          43573 │               12528 ║
+║ simple - diff from previous    │    11 │               │          43724 │               12679 ║
```

Comment:
The gas increase seems fairly consistent.  Do we know what caused it?  The switch to _validateDeadline doesn't seem that wasteful.

Reply:
I think we've crossed some glass ceiling in the stack, and the compiler (we use via-ir) now automatically uses memory (which is more expensive).

Reply:
> I think we've crossed some glass ceiling in the stack
no. that's not the case. the code compiles without via-ir. 
its the cost of bubbling up return values from inner method.
see latest commit (pass `opinfo` instead of `info.mUserOp.paymaster` into the method, and it saves 30 gas for each entry.